frontend/views/components/Badge.vue

```diff
@@ -1,12 +1,16 @@
 <template lang="pug">
-span.c-badge(v-if='number') {{number}}
+span.c-badge(:class="`is-${type}`")
+  slot
 </template>
 
 <script>
 export default {
   name: 'Badge',
```

Comment:
It would be nice if you added this component to our Design System page as well! [http://localhost:8000/app/design-system](http://localhost:8000/app/design-system)

Reply:
I will definitely work on it. :)

Reply:
Looks cool, well done!